src/app/core/services/auth/auth.service.ts

```diff
@@ -36,8 +38,12 @@ export class AuthService {
 
     this.afAuth
       .signInWithEmailAndPassword(email, password)
-      .then((credential) => {
+      .then(async (credential) => {
         this.message = '';
+        const isPostulantExist = await this.postulantsService.getIsPostulantExist(email)
```

Comment:
Trusting in promises specially with `await` blocks the js event loop, with my suggestion there's no need to do that because you are still relying on observables and reusing the functionality that it's already there